test/blackbox-tests/test-cases/pkg/ocamllsp/dev-tool-ocamllsp-env-path-var.t

```diff
@@ -0,0 +1,35 @@
+Test that the ocamllsp dev tool executes in an environment where other dev
+tools are in PATH.
+
+  $ . ../helpers.sh
+  $ . ./helpers.sh
+
+  $ mkrepo
+  $ mkpkg ocaml 5.2.0
+  $ setup_ocamllsp_workspace
+
+Make a fake ocamllsp package that prints out the PATH variable:
```

Comment:
Needing this workaround for testing seems to me like a good indication that we should just have a way of doing `dune tools exec -- foo`, so we could do `dune tools exec -- echo $PATH` That can be added as a followup. I assume it would be pretty straightforward. To you agree!

In any case, I think a TODO here to note that we should replace this workaround in the future would make sense.